Libs/DICOM/Core/ctkDICOMDatabase.cpp

```diff
@@ -3070,7 +2970,8 @@ void ctkDICOMDatabase::updateDisplayedFields()
   Q_D(ctkDICOMDatabase);
 
   // Get the files for which the displayed fields have not been created yet (DisplayedFieldsUpdatedTimestamp is NULL)
-  //TODO: handle cases when the values actually changed; now we only cover insertion and schema update
+  // Note: The per-instance update only covers insertion and schema update. If fields on the series/study/patient level need to be
+  // updated on the insertion of a new instance, then it can be handled using the startUpdate/endUpdate functions of the roles.
```

Comment:
```suggestion
  // updated on the insertion of a new instance, then it can be handled using the startUpdate/endUpdate functions of the rules.
```

Reply:
Apparently I made this mistake at many places, including the branch  name and commit message. I fixed them all except for the branch name.